test/BuildingRegistry.Tests/ProjectionTests/Wfs/BuildingV2Tests.cs

```diff
@@ -492,6 +499,132 @@ await Sut
                 });
         }
 
+        [Fact]
+        public async Task WhenBuildingMergerWasRealized()
+        {
+            var @event = _fixture.Create<BuildingMergerWasRealized>();
+
+            await Sut
+                .Given(
+                    new Envelope<BuildingMergerWasRealized>(
+                        new Envelope(
+                            @event,
+                            new Dictionary<string, object> { { AddEventHashPipe.HashMetadataKey, @event.GetHash() } })))
+                .Then(async ct =>
+                {
+                    var item = await ct.BuildingsV2.FindAsync(@event.BuildingPersistentLocalId);
+                    item.Should().NotBeNull();
+
+                    item.Status.Should().Be(BuildingV2Projections.MapStatus(BuildingStatus.Realized));
+                    var wkbReader = WKBReaderFactory.Create();
+                    var polygon = wkbReader.Read(@event.ExtendedWkbGeometry.ToByteArray());
+                    item.Geometry!.AsBinary().Should().BeEquivalentTo(polygon.AsBinary());
+                    item.GeometryMethod.Should().Be(GeometrieMethode.IngemetenGRB.ToString());
+
+                    item!.Version.Should().Be(@event.Provenance.Timestamp);
+                });
+        }
+
+        [Fact]
+        public async Task WhenBuildingWasMerged()
+        {
+            var buildingWasPlanned = _fixture.Create<BuildingWasPlannedV2>();
+            var @event = _fixture.Create<BuildingWasMerged>();
+
+            await Sut
+                .Given(
+                    new Envelope<BuildingWasPlannedV2>(
+                        new Envelope(
+                            buildingWasPlanned,
+                            new Dictionary<string, object> { { AddEventHashPipe.HashMetadataKey, buildingWasPlanned.GetHash() } })),
+                    new Envelope<BuildingWasMerged>(
+                        new Envelope(
+                            @event,
+                            new Dictionary<string, object> { { AddEventHashPipe.HashMetadataKey, @event.GetHash() } })))
+                .Then(async ct =>
+                {
+                    var item = await ct.BuildingsV2.FindAsync(@event.BuildingPersistentLocalId);
+                    item.Should().NotBeNull();
+
+                    item.Status.Should().Be(BuildingV2Projections.MapStatus(BuildingStatus.Retired));
+                    item!.Version.Should().Be(@event.Provenance.Timestamp);
+                });
+        }
+
+        [Fact]
+        public async Task WhenBuildingUnitWasTransferred()
+        {
+            var buildingWasPlannedV2 = _fixture.Create<BuildingWasPlannedV2>();
+
+            var oldBuildingPersistentLocalId = new BuildingPersistentLocalId(2);
+            var buildingUnitPersistentLocalId = new BuildingUnitPersistentLocalId(3);
+
+            var @event = new BuildingUnitWasTransferred(
```

Comment:
I've added an BuildingUnitWasTransferredBuilder in the last commit. You can use this one?